package.json

```diff
@@ -59,9 +59,13 @@
     "autoprefixer": "9",
     "classnames": "2.3.1",
     "craco-alias": "3.0.1",
+    "cypress": "9.0.0",
+    "cypress-dotenv": "^2.0.0",
+    "dotenv": "^10.0.0",
```

Comment:
fix version `"dotenv": "10.0.0",`
an then run `yarn install` to fix at yarn.lock